src/renderer/hooks/usePaneNodeSearchMenu.tsx

```diff
@@ -447,7 +450,9 @@ export const usePaneNodeSearchMenu = (
     ));
 
     const onConnectStart = useCallback(
-        (event: React.MouseEvent, handle: OnConnectStartParams) => {
+        (event: React.MouseEvent | React.TouchEvent, handle: OnConnectStartParams) => {
+            // eslint-disable-next-line no-param-reassign
+            event = event as React.MouseEvent;
```

Comment:
I think it would be better to support touch events instead of assuming that they'll never happen (can they happen?). In theory, we could just average all touchpoints and use that.

Reply:
Sure i could try that. I just assumed touch would never be use for this (especially since I'm pretty sure we don't have any of the necessary react flow stuff set up that would allow touch dragging

Reply:
Hmmm, then maybe we (read you) should the RF guys about this?

Reply:
Tbh I think handling touch stuff should be in a different PR. I don't see a reason to not merge this just because suddenly we wanna support touch devices

Reply:
Yeah, you're right.